flake.nix

```diff
@@ -100,15 +100,9 @@
     {
       packages = {
         default = scopeHost.main;
```

Comment:
We should possibly also make this `scopeHostStatic.main`, since dynamically-linked jemalloc builds are currently broken. We should either do this or fix dynamic jemalloc before the release, otherwise we'll break things for people using the `default` output in nix. There was at least one person in the matrix room that was using it.

Reply:
I made this `scopeHostStatic` anyways since from my understanding this is a better default. If not, I can revert that.